morse_code.rb

```diff
@@ -0,0 +1,52 @@
+MORSE_CODE = {
+  'A' => '.-',
+  'B' => '-...',
+  'C' => '-.-.',
+  'D' => '-..',
+  'E' => '.',
+  'F' => '..-.',
+  'G' => '--.',
+  'H' => '....',
+  'I' => '..',
+  'J' => '.---',
+  'K' => '-.-',
+  'L' => '.-..',
+  'M' => '--',
+  'N' => '-.',
+  'O' => '---',
+  'P' => '.--.',
+  'Q' => '--.-',
+  'R' => '.-.',
+  'S' => '...',
+  'T' => '-',
+  'U' => '..-',
+  'V' => '...-',
+  'W' => '.--',
+  'X' => '-..-',
+  'Y' => '-.--',
+  'Z' => '--..',
+  '0' => '-----',
+  '1' => '.----',
+  '2' => '..---',
+  '3' => '...--',
+  '4' => '....-',
+  '5' => '.....',
+  '6' => '-....',
+  '7' => '--...',
+  '8' => '---..',
+  '9' => '----.'
+}.freeze
+
+def decode_char(char)
+  Morse_code.key(char)
+end
```

Comment:
- [ ] Your code is not running properly. It shows the following error. You can try Morse_code to MORSE_CODE to fix the issue. :+1: 
![morsse](https://github.com/Rana-Mudassir/decode_morse_code/assets/70483835/a98ecac9-703b-47d1-88c7-8abee5d08303)

Reply:
Oh, thanks you @mahbubul14